bindings/python/justfile

```diff
@@ -1,29 +1,25 @@
 set fallback
-VENV := env_var_or_default("VENV", "../../.venv")
+VENV := env_var_or_default("VENV", ".venv")
 VENV_BIN := VENV / "bin"
 
 @venv:
-  python3 -c "import virtualenv" || python3 -m pip --quiet install virtualenv
-  python3 -m virtualenv {{VENV}} --quiet
-
-## Set up virtual environment and install requirements
-@requirements: venv
-  {{VENV_BIN}}/python -m pip --quiet install --upgrade pip
-  {{VENV_BIN}}/pip --quiet install -r requirements.txt
+  python3 -m venv {{VENV}}
```

Comment:
`virtualenv` was used because it has support for more terminals.

Reply:
is that material in this case where we're just calling the paths' directly with these targets rather than using he `activate` script?

Reply:
It does force everything to go through the justfile which is mildly annoying. I  `activate` and run things locally a lot so this disrupts my current workflow. I'd prefer to avoid that disruption.

Reply:
If using `virualenv`, can we install it via `venv` first. This works on MacOS for me:

```shell
# Installs python dependencies for testing
venv:
  python3 -m venv .venv_temp
  ./.venv_temp/bin/pip3 install virtualenv
  ./.venv_temp/bin/python3 -m virtualenv {{VENV}}
  {{VENV_BIN}}/python -m pip install --upgrade pip
  {{VENV_BIN}}/python -m pip install poetry
```

Reply:
I think I accidentally complicated matters.

I do need a path forward though as I currently am having trouble even running `just pytest`...whatever gets me there without interference of others' workflows is good enough

Reply:
Because the virtualenv helper doesn't matter functionally, and because @greyscaled's issue was preexisting, I've made this conditional (so you get `virtualenv` if you have it installed and otherwise you get `venv`). I think we should address Grey's issue/environment separately...

This patch is meant to fix the issue with the implicit dependencies in the pypi package, and I think it does a decent job of that.